packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -103,21 +266,30 @@ export default function TaskQuickAssignModal({
       }
       icon={<Person />}
     >
-      <ReactSelect
-        data-cy="quickAssign-assignee"
-        defaultValue={selectedWorker}
-        label={t('ADD_TASK.ASSIGNEE')}
-        options={options}
-        onChange={setWorker}
-        style={{ marginBottom: '24px' }}
-        isSearchable
+      <Controller
+        control={control}
+        name={ASSIGNEE}
+        render={({ field: { onChange } }) => (
+          <ReactSelect
+            data-cy="quickAssign-assignee"
+            defaultValue={selectedWorker}
```

Comment:
Shouldn't ReactSelect automatically update the default option when selectedWorker is set in react hook form?

Reply:
I found a way to do it, thank you!